### ISSUE TITLE
defaultValue not visible

### ISSUE THREAD

**Issue by danilobuerger** [marmelab/admin-on-rest#727]
I have something like:

```
  <Create {...props} title="Create">
    <SimpleForm>
      <DisabledInput source="id" defaultValue={() => uuid()} />
    </SimpleForm>
  </Create>
```

This gets rendered as:

<img width="337" alt="screen shot 2017-06-04 at 00 32 52" src="https://cloud.githubusercontent.com/assets/996231/26757498/642cca88-48bd-11e7-982f-20e160be4162.png">

However, I would expect it to show the uuid instead.

This seems similar to #384

**Comment by wesley6j** [comment]
Because `DisabledInput` tries to get value from record which is an empty object in `Create`. So it appears that this value is empty.

But if you submit the form, you will find that uuid is among the list.

**Comment by danilobuerger** [comment]
Yes, that's true. However, I would not consider that expected behavior.

**Comment by fzaninotto** [comment]
Fixed by #729
